zppy/templates/default.ini

```diff
@@ -188,7 +188,10 @@ run_type = string(default="model_vs_obs")
 # "qbo" requires `ref_final_yr` to be set.
 # "streamflow" requires `streamflow_obs_ts` to be set.
 # "tc_analysis" requires `tc_obs` to be set.
-sets = string_list(default=list("aerosol_aeronet","aerosol_budget","annual_cycle_zonal_mean","cosp_histogram","lat_lon","meridional_mean_2d","polar","zonal_mean_2d","zonal_mean_2d_stratosphere","zonal_mean_xy"))
+# The order of the `sets` list is the order the sets will show up in E3SM Diags.
```

Comment:
I kept the comments above listing sets in alphabetical order, for ease of readability.